modules/cnf-about-ptp-and-clock-synchronization.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-about-ptp-and-clock-synchronization_{context}"]
+= About PTP and clock synchronization error events
+
+Cloud native applications such as virtual RAN require access to notifications about hardware timing events that are critical to the functioning of the overall network. Fast event notifications are early warning signals about impending and real-time PTP clock synchronization events. PTP clock synchronization errors can negatively affect the performance and reliability of your low latency application, for example, a vRAN application running in a distributed unit (DU).
```

Comment:
PTP acronym undefined. Consider spelling out `Precision Time Protocol` when it first appears.